examples/profile.html

```diff
@@ -19,8 +19,8 @@
   </head>
   <body ng-controller="MainController as ctrl">
     <div class="profile" ngeo-profile="::ctrl.profileData"
-         ngeo-profile-onhover="ctrl.onProfileHover(point)"
-         ngeo-profile-onout="ctrl.onProfileOut()"></div>
+         ngeo-profile-options="::ctrl.profileOptions">
```

Comment:
You can remove the one-time binding expression here if you remove the "options" watcher.